configs/test_config.yml

```diff
@@ -40,6 +47,24 @@ cut_variation:
             min: [0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.06, 0.04, 0.04]
             max: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.60, 0.60]
             step: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
+        skip_cuts: [    # index of the specific cuts to skip for each pt bin
```

Comment:
this has to be added in a dedicated entry in the config file for the fraction otherwise it will be confusing